packages/jest-diff/src/diff_strings.js

```diff
@@ -207,6 +322,77 @@ const getterForHunks = (original: Original) => {
   };
 };
 
+// Return index at end of sequence of lines which correspond to a chunk.
+const getIndexAfterDiffChar = (
+  lines: Array<string>,
+  char: string,
+  i: number,
+): number => {
+  while (i !== lines.length && lines[i][0] === char) {
```

Comment:
`return lines.find(line => line[o] !== char)` instead of the `while`, right?

Reply:
No, you're looking for the index. nvm then 🙂

Reply:
Yeah, more `while` loops that I wrote in a while.

Important here to return initial index if line does not have that char or if at end of array.

Reply:
On second thought, what I meant to say is I would have followed more fluent JavaScript idiom to return `-1` if not found, except that `findIndex` method does not have a `fromIndex` argument like `indexOf` method. Therefore, this C language style of parsing loop instead.